src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -413,7 +413,7 @@
     <value>In most programming languages there is no default implementation of the equality operator (==) for value types. If your programming language supports operator overloads, you should consider implementing the equality operator. Its behavior should be identical to that of Equals.</value>
   </data>
   <data name="OverloadOperatorEqualsOnOverridingValueTypeEqualsMessage" xml:space="preserve">
-    <value>Implement the equality operators and make their behavior identical to that of 'Equals' method</value>
+    <value>Implement the equality operators and make their behavior identical to that of the Equals method</value>
```

Comment:
I don't have a strong opinion on the way forward but as far as I know the idea is to always put the single quotes when the word refers to a type, method... so I would stick to `the 'Equals'`.

Cc @mavasani

Reply:
I agree with quotes for things that are code but not for proper names. 'Equals' could be either depending on how it is used in the message. In "the Equals method," it's being used as a proper name and not as a quote of C# syntax as such.

Reply:
I am not a native English speaker so I won't make any decision on that but I think I would have _wrongly_ written "identical to that of the `Equals` method." xD

Anyways @mavasani will review and give his opinion here.

Reply:
> This is my reasoning: if you were writing about it here, you would say either "identical to that of the Equals method" or "identical to that of Equals." Not "identical to that of the Equals method."

Seems reasonable to me. @Evangelink is correct that we try to put actual symbol names in quotes, but this change seems fine.